llvm/test/Transforms/InstCombine/load-cmp.ll

```diff
@@ -335,16 +328,12 @@ define i1 @test10_struct_arr_noinbounds_i64(i64 %x) {
   ret i1 %r
 }
 
-@table = internal constant [2 x ptr] [ptr @g, ptr getelementptr (i8, ptr @g, i64 4)], align 16
+@table = internal constant [2 x ptr] [ptr @g, ptr getelementptr inbounds (i8, ptr @g, i64 4)], align 16
```

Comment:
This is a workaround for https://github.com/llvm/llvm-project/pull/98339#discussion_r1672557022.